src/site/stages/build/drupal/graphql/landingPage.graphql.js

```diff
@@ -5,7 +5,16 @@
 module.exports = `
   fragment landingPage on NodeLandingPage {
     entityUrl {
-      path
+      ... on EntityCanonicalUrl {
```

Comment:
Maybe this should be a fragment?

Reply:
I like to use fragments to share those fields across queries, but it's not required.